packages/eslint-plugin-putout/lib/putout-wrapper.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+module.exports = () => {
+    try {
+        const putout = require('putout');
+        return putout;
+    } catch (e) {
+        throw e;
+    }
```

Comment:
I think there is no need in separate file `putout-wrapper`, and let's use more user friendly message :).
Something like, `Can't load putout, please install it: ${e.message}`.

Also would be amazing to add test for this. You can use `mock-require` for this purpose, similar to [packages/putout/lib/cli/eslint.spec.js](https://github.com/coderaiser/putout/blob/v11.0.0/packages/putout/lib/cli/eslint.spec.js).